.github/workflows/release-desktop-winms.yml

```
@@ -98,17 +97,33 @@ jobs:
-          name: onekey-desktop-windows-${{ matrix.arch }}
+          name: onekey-desktop-winms-x64-exe
+          path: |
+            ./apps/desktop/build-electron/*-x64.exe
+
+      - name: Upload arm64 Windows Store exe
+        uses: actions/upload-artifact@v4
+        with:
+          name: onekey-desktop-winms-arm64-exe
+          path: |
+            ./apps/desktop/build-electron/*-arm64.exe
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)